### ISSUE TITLE
Birthday

### ISSUE THREAD

**Issue by cdesch** [railslove/birthday#11]
I'm interested in taking over maintenance.

**Comment by styrken** [comment]
Yeah i would like to help as well if needed

**Comment by bumi** [comment]
please send pull requests. I am happy to merge an release new versions. 
But for security reasons I can not directly give you access to the gem.

**Comment by styrken** [comment]
Are you sure? I maintain other libs and have access. Anyway, I see two open PRs. Maybe they should either be merged og rejected first.

**Comment by bumi** [comment]
sorry for my short answer before. 
first of all THANKS for the interest to help this gem!! that is super cool and this gem desperately needs somebody to maintain it.
As you mentioned already the two PRs must be reviewed and answered and certain updates need to be made. 

I am just hesitant to immediately transfer the ownership of the repository and the rubygem to somebody new. Applications depend on the gem and there have been instances of hijacked gems before that introduced malicious code. - just saying this as a very general thought and precaution and I hope you understand that. 

But imo in open source development the status "owner" of the repo or rubygem does not mean much and is only needed to officially release the gem. You can do anything you think is important for the gem.

Can you help reviewing the two open PRs? 
Please feel free to do anything you think is important for the gem. You have my full support and just let me know if you need anything. 

Thanks again for your support and I am super happy to see that the development of this nice gem has a chance to continue.

**Comment by cdesch** [comment]
Sure. What else needs to be done?

**Comment by cdesch** [comment]
before the pull requests, I think some general updating needs to be done. For Example, I think dropping support for the [mysql](https://github.com/luislavena/mysql-gem) gem would be good. That one hasn't been updated in 7 years.

So, for now, I'm going through everything to get things to run first

**Comment by bumi** [comment]
nice! yeah, sounds great! 
general updating is for sure the first helpful thing.

**Comment by styrken** [comment]
Yeah makes sense. I though you could get merge / release access without being owner. :-)

@cdesch Is there a slack or other place we coordinator ? (or maybe just use the projects here directly in GitHub)

**Comment by cdesch** [comment]
I can create one. I’ll come up with a list or stuff todo today.

**Comment by styrken** [comment]
Ok, sounds great

**Comment by styrken** [comment]
@cdesch did you ever create a list? :-)
